libs/ml/include/ml/dataloaders/word2vec_loaders/vocab.hpp

```diff
@@ -42,6 +57,101 @@ class Vocab
   SizeType    IndexFromWord(std::string const &word) const;
 };
 
+Vocab::Vocab()
+{}
+
+/**
+ * Update the data in vocabulary every time vocabulary is changed
```

Comment:
i guess you mean the 'meta-data'? since that's whats being updated here

Reply:
what do u mean by meta-data?